### PR TITLE
W-12980682: Update groovy used at build time to 4.x

### PR DESCRIPTION
**Pull request by elrodro83**

### CODE REVIEW

File: embedded/embedded-distribution/pom.xml [hunk 1]
```diff
@@ -37,19 +37,29 @@
...
                     <execution>
                         <phase>package</phase>
                         <goals>
                             <goal>execute</goal>
                         </goals>
                         <configuration>
-                            <source>${project.basedir}/BuildAggregator.groovy</source>
+                        	<scripts>
+                                <script>file:///${project.basedir}/BuildAggregator.groovy</script>
+                        	</scripts>
```
Comment (asanguinetti) [inline embedded/embedded-distribution/pom.xml:51]: There seems to be a problem with tabs vs spaces here.

**Review by asanguinetti — CHANGES_REQUESTED** [review]
Formatting issue

**Review by asanguinetti — APPROVED** [review]

**Comment by elrodro83** [comment]
--validate

**Review by asanguinetti — DISMISSED** [review]

**Comment by elrodro83** [comment]
--validate

**Comment by elrodro83** [comment]
--validate

**Pull request merged**